rate-limiting.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Rate limiting
+description: "What is rate limiting and how to deal with it."
+icon: "gauge-max"
+---
+
+Whenever the octomind agent is discovering your page or the test runner is executing test cases it is sending requests requests to your page (or your app).
+
+To ensure a well defined behavior all requests are send through one fixed IP address: `35.192.162.70`
```

Comment:
can we just link to the section with the IP so we don't duplicate it? Also there's 2 ;)

Reply:
I will do another PR and I'll add it here.